src/components/SignUpSuccessForm/index.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+
+const SignUpSuccessForm = () => {
+  return (
+    <section className="sign-up-success-form">
+      <div className="sign-up-success-form__image">Sample Picture</div>
+      <header className="sign-up-success-form__header">
+        <h2>Registration Success</h2>
+      </header>
+      <footer className="sign-up-success-form__footer">
+        <div className="sign-up-success-form__actionable">
+          <button className="sign-up-success-form__button">Continue</button>
+        </div>
+      </footer>
```

Comment:
I think we don't need the footer though whereas we can add properties in `__actionable`? 🤔